build.gradle.kts

```diff
@@ -199,7 +199,7 @@ tasks {
     return destinationDir
   }
 
-  val codyCommit = "70e927ce69c75aea5b2053574333e6525b717ba3"
+  val codyCommit = "ef011c45c0b28446aaf5921e8080b4492469b5fb"
```

Comment:
Is it required for this PR? If not i thought that we should update cody commit in separate PRs.